### PR TITLE
TST: fix complicated test on old python

### PR DESCRIPTION
**Pull request by mattip**
Fixes the fix to gh-15170 in gh-15187 for python3.5

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]

File: numpy/random/tests/test_extending.py [hunk 1]
```diff
@@ -43,8 +43,8 @@
 @pytest.mark.skipif(sys.platform == 'win32', reason="cmd too long on CI")
```
Comment (charris) [inline numpy/random/tests/test_extending.py]: Do we still need to skip the test on windows?

Comment (charris) [reply numpy/random/tests/test_extending.py]: I'd like to just close the backport, it should be combined with this PR.

Comment (eric-wieser) [reply numpy/random/tests/test_extending.py]: Should be harmless to merge this one, and then close / reopen the backport

Comment (charris) [reply numpy/random/tests/test_extending.py]: Seems overly complicated for a one line change. Could just edit the other PR.

Comment (mattip) [reply numpy/random/tests/test_extending.py]: ok, lets do it that way

**Pull request closed without merging**